portfolio/src/main/webapp/script.js

```diff
@@ -12,13 +12,84 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(init);
+
+    function init() {
+        //raw data
+        var languages = [['Language', 'Proficiency Level', { role: 'style' }, { role: 'annotation' } ],
+                        ['Java', 5, 'color: #006400', 'Advanced' ],
```

Comment:
For readability, is it possible to use a datastructure for these colors. How I imagine it would be like is `var colors =  {"colorName1" : "#colorvalue1","colorName2" : "#colorvalue2"}` and you can just refer to a color like `colors["colorName1"]`. 

It makes it more obvious to someone reading this code what the color is, and if you instead decide to use specific naming forkeys, it makes it a little easier to change your color theme since you just have to update that data structure instead of finding all times you use a color value.

Reply:
fixed!